src/Microsoft.Extensions.Logging.Abstractions/NullLoggerFactory.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.Abstractions
+{
+    /// <summary>
+    /// An <see cref="ILoggerFactory"/> used to create instance of
+    /// <see cref="NullLogger"/> that logs nothing.
+    /// </summary>
+    public class NullLoggerFactory : ILoggerFactory
+    {
+        public static readonly NullLoggerFactory Instance = new NullLoggerFactory();
+
+        /// <inheritdoc />
+        /// <remarks>
+        /// This returns a <see cref="NullLogger"/> instance which explicitly logs nothing.
+        /// </remarks>
+        public ILogger CreateLogger(string name)
+        {
+            return NullLogger.Instance;
+        }
+
+        /// <inheritdoc />
+        /// <remarks>
+        /// This method explicitly no ops.
```

Comment:
I think this phrasing can be improved, `no ops` is nerd-speak. How about "This method ignores the parameter and does nothing."